examples/mosaic-integration/src/types.d.ts

```diff
@@ -81,6 +85,9 @@ declare module '@uwdata/vgplot' {
     bottom: number;
   }): Directive;
   function marginLeft(value: number): Directive;
+  function marginTop(value: number): Directive;
+  function marginLeft(value: number): Directive;
```

Comment:
`marginLeft` is duplicated.

Also, best to keep these alphabetized, at least in each section. Helps to catch duplicates liket his.

Reply:
Good catch, thanks!